context/app/static/js/components/entity-search/SearchWrapper/utils.js

```diff
@@ -42,7 +42,7 @@ function buildFieldConfig({
   type,
   facetGroup = 'Additional Facets',
   configureGroup = 'General',
-  entityType: pageEntityType,
+  entityType: _pageEntityType,
```

Comment:
Since this prop is part of a spread deconstruction operator, removing it altogether would implicitly add `entityType` to the `rest` object whenever  `entityType` is ever provided as part of the input object. This would lead to the `entityType` key being propagated into the field config returned since `...rest` is part of the returned fields, and may cause unforeseen bugs to pop up. My understanding of this key being present in the deconstructed props but unused otherwise is that `entityType` is being purposely excluded from the returned object (assuming that the related search functionality is working as expected)